.circleci/config.yml

```diff
@@ -13,14 +13,17 @@ commands:
     description: 'Save the Bundler cache'
     steps:
       - save_cache:
-          key: 'v2-ruby-dependencies-{{ arch }}-{{ checksum "Gemfile.lock" }}'
+          key: &bundler-key >-
```

Comment:
Please change to `bundler-cache-key`, as this signifies _exactly_ what it's for.